osu.Game/Rulesets/Mods/ModMuted.cs

```diff
@@ -101,4 +103,9 @@ public void ApplyToScoreProcessor(ScoreProcessor scoreProcessor)
 
         public ScoreRank AdjustRank(ScoreRank rank, double accuracy) => rank;
     }
+
+    public class MuteComboSlider : OsuSliderBar<int>
+    {
+        public override LocalisableString TooltipText => Current.Value == 0 ? "(always muted)" : base.TooltipText;
```

Comment:
why is it in brackets?

Reply:
Wasn't sure how to go with displaying this, `always muted`, `(always muted)`, `0 (always muted)`. Not sure if there's a standard way I failed to notice, but I'll go with `always muted` then.